src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figure = new Figure[6];
+        for (int i = 0; i < figure.length; i++) {
+            if (i < figure.length / 2) {
+                figure[i] = new FigureSupplier().getRandomFigure();
+            } else {
```

Comment:
Don’t complicate if-else construction. Check [this](https://www.youtube.com/watch?v=P-UmyrbGjwE&list=PL7FuXFaDeEX1smwnp-9ri8DBpgdo7Msu2&ab_channel=Mateacademy) please.

Reply:
I can realize here the way without if-else construction, but as for me it will be less optimized rather than first variant. 
I mean I can realize a few loops: 1 - with call getRandomFigure, 2 - with call getDefaultFigure and 3 - print all figures; it`ll no need the condition, but need more code.